### ISSUE TITLE
Cannot use pending updates while swapping plans

### ISSUE THREAD

**Issue by driesvints** [laravel/cashier-stripe#969]
- Cashier Version: 12.0.0

Atm Cashier cannot use pending updates while swapping plans due to a bug in Stripe which prevents pending updates while deleting subscription items.

Reproducable by:

```php
$subscription->pendingIfPaymentFails()->swap('new-plan');
```

This will delete the old subscription item and create a new one. Deleting subscription items isn't available atm in Stripe. I'm in contact with Stripe and following up.

**Comment by driesvints** [comment]
Reminder to self: re-send https://github.com/laravel/docs/pull/6090 when this is resolved.

**Comment by driesvints** [comment]
Stripe let me know that they'd backlogged a fix for this so pending updates is unfortunately not going to happen atm for Cashier.

**Comment by Sans84** [comment]
@driesvints 
Now using the Stripe API I can swap the plan using a pending update. But I still can't use $subscription->pendingIfPaymentFails()->swap('new-plan'). Now I get the error: "When `payment_behavior` is set to `pending_if_incomplete`, you can only pass supported params. `items[0][tax_rates]` is not supported."

**Comment by driesvints** [comment]
Unfortunately not... I filtered out the `tax_rates` from the request but I still get the error from this issue:

> Stripe\Exception\InvalidRequestException: When `payment_behavior` is set to `pending_if_incomplete`, you can only pass supported params. `items[1][deleted]` is not supported.

**Comment by jamesh-purr** [comment]
Hey @driesvints 

Why was this issue closed? The issue still exists when trying to use pending payments on subscription updates?

**Comment by jamesh-purr** [comment]
Hey @driesvints 

Thanks for getting back to me. I got this from stripe support which appears to be in their api?

https://stripe.com/docs/billing/subscriptions/pending-updates#canceling-changing

It just seems that updating subscription itself via api should be fine. The issue regarding tax rates I ran into just like above. 

Would that not suggest this is an issue with the code structure/the way updates are processed in cashier than a stripe api issue? Apologies if I am misunderstanding what you mean by a stripe api limitation.
